<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup_PdusQueryDlg</class>
 <widget class="QDialog" name="Setup_PdusQueryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="roomEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>机房：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QComboBox" name="devtypeCmb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>181</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>所有类型</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SI-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R-PDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MPDU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ZPDU</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入查询信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设备类型：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="devnumCmb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>所有</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>主机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副机9</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设备IP：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cabEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>级联：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>机柜：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
